### ISSUE TITLE
List Games

### ISSUE THREAD

**Issue by oestrich** [oestrich/grapevine-mobile#1]
The start screen of the application should list all live games. This is fetched via:

```curl https://grapevine.haus/games -H "Accept: application/json"```

There are pagination links at the bottom that let you fetch the rest of the collection. 20 results are returned at a time.

For start, the name and tagline in a list view is enough.

**Comment by pbfrias17** [comment]
is there an api for this? fetching gives me html

**Comment by oestrich** [comment]
Yeah, you need to add the accept header to get JSON back. That curl command should work.
